handler/src/main/java/io/netty/handler/ssl/JdkAlpnApplicationProtocolNegotiator.java

```diff
@@ -130,6 +132,9 @@ public SSLEngine wrapSslEngine(SSLEngine engine, ByteBufAllocator alloc,
                 return isServer ? JettyAlpnSslEngine.newServerEngine(engine, applicationNegotiator)
                         : JettyAlpnSslEngine.newClientEngine(engine, applicationNegotiator);
             }
+            if (Java9SslUtils.supportsAlpn()) {
```

Comment:
in regard to the above, can you move this above the `if (JettyAlpnSslEngine.isAvailable()) {` ?  Jetty is a horrible hack and should be avoided if possible.